h/services/organization.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+from h.models import Organization
+
+
+class OrganizationService(object):
+
+    """A service for manipulating organizations."""
+
+    def __init__(self, session):
+        """
+        Create a new organizations service.
+
+        :param session: the SQLAlchemy session object
+        """
+        self.session = session
+
+    def create_organization(self, name, authority, logo=None):
+        """
```

Comment:
Was there a particular motivation for creating a service to handle organization creation, as opposed to just creating the `Organization` directly? If there is additional logic that we expect to have beyond passing params to the `Organization` constructor then it would make sense.

Reply:
I suppose you could create an org directly in the cli. I was following the design pattern that was already there in that the cli command calls into the service. Also there will be a need to delete an organization and rename/update the logo of an organization at some point in the near future. I was thinking that the admin panel would be able to use this service as well but I don't know much about it yet so I could be wrong there.

Reply:
I could see arguments for having the service here for consistency's sake (keeping direct interaction with models abstracted from other pieces of the architecture).

What do you think about naming this method `create` instead of `create_organization`? The `group` service has more granularly-named methods for create because there are several "types" of group, but the `user` service, e.g., has simpler naming (`fetch`, e.g. vs. `fetch_user`).

Reply:
>  Also there will be a need to delete an organization and rename/update the logo of an organization at some point in the near future.

Good point, also.